setup.py

```diff
@@ -278,6 +278,48 @@ def __is_using_gnu_linker(self, ext):
                     is_gcc = "gcc" in compiler_name or "g++" in compiler_name
             return is_gcc and sysconfig.get_config_var('GNULD') == 'yes'
 
+    # Overriding due to https://github.com/numpy/numpy/issues/12530, which
+    # is showing up in Azure CI as:
+    #         ValueError: path is on mount 'c:', start on mount 'D:'
+    def _process_unlinkable_fobjects(self, objects, libraries,
```

Comment:
Note that this is just function definition (see the indentation)... it doesn't actually override anything